packages/web-app-files/src/views/trash/Overview.vue

```diff
@@ -175,7 +174,27 @@ export default defineComponent({
     ])
 
     const getSpaceName = (space: SpaceResource) => {
-      return isPersonalSpaceResource(space) ? $gettext('Personal') : space.name
+      if (isPersonalSpaceResource(space)) {
+        return $gettext('Personal')
+      }
+      if (space.disabled !== false) {
+        return `${space.name} (${$gettext('disabled')})`
```

Comment:
```suggestion
        return $gettext('%{spaceName} (disabled)', { spaceName: space.name })
```

Please put as much as possible into transifex, always. 🙈  There are rare cases where concatenation of translated strings is not avoidable. But most of the time it is possible and should be the preferred solution. Translations are a complicated topic, this makes it a little bit easier.